docs/_content/ontwikkelaars/handleidingen-en-tutorials/large-files.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "Large files upload tutorial"
+weight: 100
+---
+
+In this tutorial we explore the ways to upload files to DRC depending on the file size. 
+
+The most relevant component for this tutorial:
+
+* DRC: for creating documents and uploading files
+
+## The requirements for this tutorial
+
+* `docker` and `docker-compose` to host the components locally on your (development machine). 
+  See [installatie en configuratie](./installatie-en-configuratie) for a detailed description.
```

Comment:
I'm not sure if this is because the PR is not merged yet, but the link doesn't work for me

Reply:
The internal links in the whole "handleidingen-en-tutorials" folder don't work for me in Github. But they work using Jekyll, and I supposed that was a point. So I made these links consistent with others. Do they work with Jekyll for you?